laba5_test.cpp

```diff
@@ -0,0 +1,81 @@
+#include "student.h"
+#include "group.h"
+
+int main()
+{
+
+    int list_count;
+    cout << "how many students in group?\t";
+    cin >> list_count;
+    vector<Student> list;
```

Comment:
'vector<Student> list;'
it would be better to give it proper name like "list_of_students"